### DISCUSSION TITLE
Docker Custom Build

### DISCUSSION THREAD

**Discussion by jadsultan123** [apache/airflow#28308] [Q&A]
### What do you see as an issue?

Hello I am getting this ERROR when building the docker image from the DockerFile in the https://github.com/apache/airflow.git repo:
Error response from daemon: dockerfile parse error line 76: unknown instruction: SET

command executed: sudo docker build . --tag custom_airflow:2.5.0

Please if you can help me fixing this issue

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by Taragolis** [comment]
You need to have installed and enabled Docker BuildKit, see documentation
https://airflow.apache.org/docs/docker-stack/build.html#customizing-the-image

**Comment by raphaelauv** [comment]
There is a nice documentation about how extend or custom the build of the container

https://airflow.apache.org/docs/docker-stack/build.html#build-build-image
